pallets/staking-rewards/src/lib.rs

```diff
@@ -380,6 +387,61 @@ pub mod pallet {
 
             InflationInfo { inflation, roi }
         }
+
+        /// The account ID of the rent rewards pool.
+        pub fn rent_pool_account_id() -> T::AccountId {
+            use sp_runtime::traits::TrailingZeroInput;
+
+            let entropy =
+                (T::PalletId::get(), b"gear rent pool").using_encoded(sp_io::hashing::blake2_256);
+            let actor_id = Decode::decode(&mut TrailingZeroInput::new(entropy.as_ref()))
+                .expect("infinite length input; no invalid inputs for type; qed");
+
+            actor_id
+        }
+
+        /// Return the amount in the rent pool.
+        // The existential deposit is not a part of the pool so the account never gets deleted.
+        pub fn rent_pool_free_balance() -> BalanceOf<T> {
+            T::Currency::free_balance(&Self::rent_pool_account_id())
+                // Must never be less than 0 but better be safe.
+                .saturating_sub(T::Currency::minimum_balance())
+        }
+    }
+
+    impl<T: Config> Get<Option<<T as frame_system::Config>::AccountId>> for Pallet<T> {
+        fn get() -> Option<<T as frame_system::Config>::AccountId> {
+            Some(Self::rent_pool_account_id())
+        }
+    }
+}
+
+fn pay_rent_rewards_out<T: Config>(maybe_active_era_info: Option<ActiveEraInfo>) {
+    let Some(active_era_info) = maybe_active_era_info else {
+        return;
+    };
+
+    let reward_points = pallet_staking::Pallet::<T>::eras_reward_points(active_era_info.index);
+    let total = u128::from(reward_points.total);
+    if total == 0 {
+        return;
+    }
+
+    let funds: u128 = pallet::Pallet::<T>::rent_pool_free_balance().unique_saturated_into();
+    for (account_id, points) in reward_points.individual {
+        let payout = funds * u128::from(points) / total;
+        if payout > 0 {
+            let result = CurrencyOf::<T>::transfer(
+                &pallet::Pallet::<T>::rent_pool_account_id(),
+                &account_id,
+                payout.unique_saturated_into(),
+                ExistenceRequirement::KeepAlive,
+            );
```

Comment:
.map_err(|e| log::error(".. {e:?}"))

Reply:
its an error because validator balance > ed && pool balance >= ed is invariant

Reply:
I'd consider addition of the era rents into era payout at all to avoid 55-inf unnecessary transfers

Reply:
Like it will motivate validators to trigger payout stakers mb

Reply:
discussed. We leave it as is for the moment and will return back to it - #3729